tests/features/save_test.py

```diff
@@ -292,15 +294,19 @@ def test_save_and_exit_to_uncreated_directory(run, tmpdir):
     assert f.read() == 'hello\n'
 
 
-def test_save_and_exit_to_dot_file(run, tmpdir):
-    f = tmpdir.join('.f')
-    with run(str(f)) as h:
-        h.press('hello')
-        h.await_text('hello')
-        h.press_and_enter('^X')
-        h.await_text('file is modified - save [yes, no]?')
-        h.press('y')
-        h.await_text('enter filename: ')
-        h.press('Enter')
-        h.await_exit()
-    assert f.read() == 'hello\n'
+@pytest.mark.parametrize('dot_file', ('.f',))
+def test_save_and_exit_to_dot_file(run, dot_file):
+    with open(dot_file, 'w+') as f:
```

Comment:
this should not write to the execution working directory -- use a filesystem fixture (`tmpdir`) and `as_cwd()` will probably do what you want